charts/rancher-monitoring/v0.2.1003/values.yaml

```diff
@@ -417,3 +417,5 @@ exporter-gpu-node:
     dcgmexporter:
       repository: cnrancher/dcgm-exporter
       tag: 1.4.6
```

Comment:
just confirm 1.4.5 is correct?

Reply:
1.4.6 is gpu monitoring v1 image version